lib/trivia_app.ex

```diff
@@ -5,18 +5,28 @@ defmodule TriviaApp do
   end
 
   @questions  [
-                %{ question: "Samite is a type of what: a) Fabric? b) Stone? c) Dog? d) Cake?", answer: "a" }, 
-                %{ question:  "Vermillion is a shade of which colour: a) Green? b) Blue? c) Red? d) Yellow?", answer: "c" }
+                %{ question: "Samite is a type of what: \na) Fabric \nb) Stone \nc) Dog \nd) Cake ", answer: "a" }, 
```

Comment:
Thank you Felipe!
Is figuring out a file format when you split the responsibilities into
seperate modules?

On Mon, Sep 5, 2016 at 4:21 PM, Felipe Seré <notifications@github.com>
wrote:

> In lib/trivia_app.ex
> <https://github.com/DaisyMolving/quiz_master_elixir/pull/1#discussion_r77536015>
> :
>
> > @@ -5,18 +5,28 @@ defmodule TriviaApp do
> >    end
> >
> >    @questions  [
> > -                %{ question: "Samite is a type of what: a) Fabric? b) Stone? c) Dog? d) Cake?", answer: "a" },
> > -                %{ question:  "Vermillion is a shade of which colour: a) Green? b) Blue? c) Red? d) Yellow?", answer: "c" }
> > +                %{ question: "Samite is a type of what: \na) Fabric \nb) Stone \nc) Dog \nd) Cake ", answer: "a" },
>
> This is pretty cool ☺️
> The next stage in "extraction" is to figure out a file format, so you
> don't even need to re-build the application to add new questions 😄👍👏
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DaisyMolving/quiz_master_elixir/pull/1/files/e20f4e75dd8bd4e797b0938517e08a2be2aaef45#r77536015>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALbNsYMuqsl-QxtgJVLLfBCGDqtRw6d9ks5qnDNtgaJpZM4J1Gmb>
> .
>